http/backup_service.go

```diff
@@ -106,6 +131,65 @@ func (h *BackupHandler) handleBackupShard(w http.ResponseWriter, r *http.Request
 	}
 }
 
+func (h *BackupHandler) handleBackupMetadata(w http.ResponseWriter, r *http.Request) {
+	span, r := tracing.ExtractFromHTTPRequest(r, "BackupHandler.handleBackupMetadata")
+	defer span.Finish()
+
+	ctx := r.Context()
+
+	baseName := time.Now().UTC().Format(influxdb.BackupFilenamePattern)
+
+	formWriter := multipart.NewWriter(w)
+	w.Header().Set("Content-Type", formWriter.FormDataContentType())
```

Comment:
Could we use `multipart/mixed` instead, to match the [API doc](https://github.com/influxdata/openapi/blob/master/src/oss/paths/backup_metadata.yml#L32)? All the descriptions I can find of `multipart/form-data` make it sounds like it's intended for uploads (i.e. [this](https://docs.microsoft.com/en-us/previous-versions/office/developer/exchange-server-2010/aa493937(v=exchg.140))).